Networking/NetworkingTests/Responses/sites-malformed.json

```diff
@@ -48,6 +48,10 @@
       "jetpack": false,
       "jetpack_connection": true,
       "was_ecommerce_trial": false,
+      "plan": {
```

Comment:
I think we also have to add this to [sites.json](https://github.com/woocommerce/woocommerce-ios/blob/trunk/Networking/NetworkingTests/Responses/sites.json) to fix the unit tests.

Reply:
Great catch, I updated the mock file in d76d619a78677baf56b3a1ceb351c3acedbb555b.